src/main/java/io/relution/jenkins/awssqs/SQSTrigger.java

```diff
@@ -77,8 +79,9 @@
     private transient ExecutorService executor;
 
     @DataBoundConstructor
-    public SQSTrigger(final String queueUuid) {
+    public SQSTrigger(final String queueUuid, final boolean disableCodeCommit) {
```

Comment:
Formally it breaks binary compatibility. It would be nice to retain the old constructor (or to use `@DataBoundSetter`)

Reply:
I retained the old constructor